x-pack/test/spaces_api_integration/common/suites/resolve_copy_to_space_conflicts.ts

```diff
@@ -327,7 +335,11 @@ export function resolveCopyToSpaceConflictsSuite(
         },
       });
     };
-    const expectSuccessResponse = (response: TestResponse, id: string, destinationId?: string) => {
+    const expectSuccessESResponse = (
```

Comment:
I would also change this function name back to `expectSuccessResponse` as this is coming from Kibana, and the same function name is used elsewhere in the API integration test suites.

Reply:
Fixed